src/common/Analytics/AnalyticsEvents.ts

```diff
@@ -0,0 +1,91 @@
+import { EventEmitter } from 'events';
+
+export type AnalyticsEventTypeHandlers = {
+  track: (eventName: string, event: unknown) => void;
+  identifyUser: (id: string) => void;
+  userPropertyUpdate: (
+    key: string,
+    value: string | number | boolean | undefined,
+  ) => void;
+  reset: () => void;
```

Comment:
What is `reset` supposed to be used for?

Reply:
It is the opposite of `identifyUser`, called when user logs out. I'll make it called `resetIdentifyUser`. I was following the library's naming but we can be more specific.

Reply:
@nicolls1 I'd suggest "resetUser"